client/scripts/views/pages/discussions/discussion_row_menu.ts

```diff
@@ -12,27 +12,28 @@ import { confirmationModalWithText } from '../../modals/confirm_modal';
 export const ThreadSubscriptionButton: m.Component<{ proposal: OffchainThread }> = {
   view: (vnode) => {
     const { proposal } = vnode.attrs;
-    const notificationSubscription = app.user.notifications.subscriptions
-      .find((v) => v.category === NotificationCategories.NewComment && v.objectId === proposal.uniqueIdentifier);
+
+    const commentAndReactionSubscriptions = app.user.notifications.subscriptions
+      .filter((v) => v.objectId === proposal.uniqueIdentifier);
+
+    const bothActive = (commentAndReactionSubscriptions[0]?.isActive && commentAndReactionSubscriptions[1]?.isActive);
```

Comment:
Because our DB doesn't allow for the creation of more than 1 of each subscription category ('new-comment-creation' or 'new-reaction') on a thread/proposal as object, I assumed that if there are two, they are one of each category. 

I can make it explicit for the sake of clarity though.